src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -16,70 +16,70 @@ public class PackageNamespacesConfiguration
 
         /// <summary>
-        /// Source name to package namespace list.
+        /// Source name to package patterns list.
         /// </summary>
-        internal Dictionary<string, IReadOnlyList<string>> Namespaces { get; }
+        internal Dictionary<string, IReadOnlyList<string>> Patterns { get; }
 
         private Lazy<SearchTree> SearchTree { get; }
 
         /// <summary>
-        /// Indicate if any packageSource exist in package namespace section
+        /// Indicate if any packageSource exist in package source mapping section
         /// </summary>
-        public bool AreNamespacesEnabled { get; }
+        public bool IsEnabled { get; }
 
         /// <summary>
-        /// Get package source names with matching prefix "term" from package namespaces section.
+        /// Get package source names with matching prefix "term" from package source mapping section.
         /// </summary>
         /// <param name="term">Search term. Cannot be null, empty, or whitespace only. </param>
-        /// <returns>Package source names with matching prefix "term" from package namespaces.</returns>
+        /// <returns>Package source names with matching prefix "term" from package patterns.</returns>
         /// <exception cref="ArgumentException"> if <paramref name="term"/> is null, empty, or whitespace only.</exception>
         public IReadOnlyList<string> GetConfiguredPackageSources(string term)
         {
             return SearchTree.Value?.GetConfiguredPackageSources(term);
         }
 
-        internal PackageNamespacesConfiguration(Dictionary<string, IReadOnlyList<string>> namespaces)
+        internal PackageSourceMapping(Dictionary<string, IReadOnlyList<string>> patterns)
         {
-            Namespaces = namespaces ?? throw new ArgumentNullException(nameof(namespaces));
-            AreNamespacesEnabled = Namespaces.Keys.Count > 0;
+            Patterns = patterns ?? throw new ArgumentNullException(nameof(patterns));
+            IsEnabled = Patterns.Keys.Count > 0;
             SearchTree = new Lazy<SearchTree>(() => GetSearchTree());
         }
 
         /// <summary>
-        /// Generates a <see cref="PackageNamespacesConfiguration"/> based on the settings object.
+        /// Generates a <see cref="PackageSourceMapping"/> based on the settings object.
         /// </summary>
         /// <param name="settings">Search term. Cannot be null, empty, or whitespace only. </param>
-        /// <returns>A <see cref="PackageNamespacesConfiguration"/> based on the settings.</returns>
+        /// <returns>A <see cref="PackageSourceMapping"/> based on the settings.</returns>
         /// <exception cref="ArgumentNullException"> if <paramref name="settings"/> is null.</exception>
-        public static PackageNamespacesConfiguration GetPackageNamespacesConfiguration(ISettings settings)
+        public static PackageSourceMapping GetPackageSourceMappingConfiguration(ISettings settings)
```

Comment:
This should be `GetPackageSourceMapping`.

Reply:
I thought about it when I renamed `PackageSourceMappingConfiguration` type to `PackageSourceMapping` but forgot to update the method name. thanks, Nikolche. Fixed in the next commit.